simple_sso/sso_client/models.py

```diff
@@ -0,0 +1,29 @@
+
+from simple_sso.settings import settings
+from django.urls import NoReverseMatch, reverse
+
+from webservices.sync import SyncConsumer
+
+
+def logout_sso_client(sender, request, user, **kwargs):
```

Comment:
Why would this receiver be in a "models" file? It does not seem to be a model/data definition. Perhaps utilize a "signals.py" file